Docs / test tools

ptomato opened the pull request:
https://phabricator.endlessm.com/T20930

ptomato committed:
Update python-dbus and python-dbusmock

I had module files lying around autogenerated by flatpak-pip-generator
for these, so I just replaced the existing definitions and added
cleanup-platform.

https://phabricator.endlessm.com/T20930

ptomato committed:
Add Hotdoc and hotdoc-modular-framework to manifest

These are required for building the modular framework documentation.
However, we don't want to build the documentation by default, only in
documentation builds.

The lxml definition and the patch are taken from org.gnome.Builder.
The lxml module is currently broken if you only download the source
package.

https://phabricator.endlessm.com/T20930

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

ptomato committed:
Remove jasmine-gjs from platform

It isn't needed at runtime, only as a developer tool for running tests.

https://phabricator.endlessm.com/T20930
